catalytic_table_to_struct/example/src/generated/child.rs

```diff
@@ -120,6 +123,7 @@ pub struct ChildRef<'a> {
     pub enum_json: &'a crate::MyJsonEnum,
     pub json: &'a crate::MyJsonType,
     pub json_nullable: &'a std::option::Option<crate::MyJsonType>,
+    pub text_nullable: &'a std::option::Option<String>,
```

Comment:
Without this change, `&'a std::option::Option<str>` would have been generated, which is not compileable